app/src/org/commcare/activities/components/ImageCaptureProcessing.java

```diff
@@ -130,6 +133,10 @@ public static void processImageChooserResponse(FormEntryActivity activity,
             showInvalidImageMessage(activity);
             return;
         }
+        if (FileUtil.isFileTooLargeToUpload(activity.getContentResolver(), selectedImage)) {
```

Comment:
we should be doing a size check on the scaled image not on the original image taken or choosen by the user.

Reply:
Oooohh right!